### DISCUSSION TITLE
Jamulus not starting because of USB micro assigned in ASIO4ALL settings

### DISCUSSION THREAD

**Discussion by waukn** [jamulussoftware/jamulus#1567] [Technical Support & Help]
I have a WIN10 laptop with all updates in place, ASIO4ALL v2.15 Beta and Jamulus. Everything works fine with the embedded "High Definition Audio Device", both for speakers and micro. If I select the very new USB micro (Jeemak), the ASIO4ALL driver accepts my selection with no errors at all. But then Jamulus gives me the message: "The audio device does not support the required sample rate. The required sample rate is 48000 Hz". The micro is much better, than required.
Help is appreciated, thanks in advance. Wilfried

![Gitta-USB-Mikro-01](https://user-images.githubusercontent.com/28543837/116214866-6d1bff00-a747-11eb-8b22-e650fb1113de.JPG)
![Gitta-USB-Mikro-02](https://user-images.githubusercontent.com/28543837/116214898-7311e000-a747-11eb-92fc-0e55ab622bcf.JPG)
![Gitta-USB-Mikro-03](https://user-images.githubusercontent.com/28543837/116214906-760cd080-a747-11eb-89eb-25b0f2ec1805.JPG)

**Comment by dcorson-ticino-com** [comment]
Apparently the USB mic is set for 44.1kHz sampling rate.  
For Jamulus you need a 48kHz sampling rate.
Look in the documentation of your mic if it is possible to change this.
Otherwise you could try the resampling option 44.1kHz - 48kHz in ASIO4ALL.

**Comment by waukn** [comment]
Thanks for the quick answer, unfortunately resampling does not work. According to the documentation the micro should support up to 192kHz. There is no option to change the sample rate to 48kHz and the driver properties have no option as well. I have ordered now another micro at Amazon, lets hope for better result.

**Comment by waukn** [comment]
I have just checked the Windows settings of the mic and it has 192 kHz (studio quality). The drop down is greyed out and therefore cannot be changed.
![Gitta-USB-Mikro-04](https://user-images.githubusercontent.com/28543837/116227615-a27b1980-a754-11eb-9194-75c5742bf146.JPG)

**Reply by henkdegroot** [reply]
It would be interesting to know which microphone you are using now. I.e. to prevent other people running into the same issue.
Also, sometimes there is "extra" software available you can download from the manufacturer which may allow access to additional settings.

**Comment by gilgongo** [comment]
Moving this to a discussion as it's less of a bug and more of a question how Jamulus might be able to handle this situation better.

**Reply by waukn** [reply]
Currently I am using the Jeemak micro, bought bei Amazon. My collegue uses a ÓTHA micro and obviously he is able to choose from a lot of different sample rates - and luckily 48kHz. I have ordered it now, will receive it tomorrow and let you all know if it works.

![Jeemak](https://user-images.githubusercontent.com/28543837/116260197-c3546680-a776-11eb-984a-76b71c27558b.JPG)

**Reply by henkdegroot** [reply]
Indeed looks like the sample rate for this microphone is fixed to 192kHz.

**Reply by waukn** [reply]
I have got a feedback from the manufacturer of the micro: You can't adjust it at all - it's fixed to 192kHz.

![Jeemak-Feedback](https://user-images.githubusercontent.com/28543837/116357744-1ffa6480-a7fd-11eb-9109-a8cf5afdf09c.JPG)
